core/tests/test_utils_test.py

```
@@ -36,9 +36,8 @@
 from core.platform import models
 from core.tests import test_utils
 
-import elasticsearch
 import webapp2
-from typing import Callable, Dict, Final, List, OrderedDict, Tuple, Union
+from typing import Callable, Final, List, OrderedDict, Tuple
```

Ruff (0.16.3)

[warning] Import from `collections.abc` instead: `Callable`

Import from `collections.abc`

(UP035)